### PR TITLE
US internships

### PR DESCRIPTION
**Pull request by shardulc**

### CODE REVIEW

File: _life/money-americans.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 layout: post
-title: "Money and Taxes Supplement for US citizens 🗽"
+title: "Money and Taxes Supplement for US-Affiliated Persons 🗽"
```
Comment (shardulc) [inline _life/money-americans.md:3]: I know this is a long and awkward title but that's kind of the joke here please humor me

Comment (SolalPirelli) [reply _life/money-americans.md:3]: go full Prince, "The people formerly known as Americans" :p (please don't)

File: _life/money-americans.md [hunk 2]
```diff
@@ -272,6 +276,38 @@ Vaud is known to unofficially grant automatic extensions until June. Two, that t
 cantons collect the federal, cantonal, and communal taxes in Switzerland, so you only
 have to file one return. Good luck!
 
+## Internships in the US for non-citizens
+
+If there is any possibility you might live in the US in the future, open an account
+at a US bank soon after you arrive for your internship. Many banks offer basic
+checking accounts with _zero fees_ if you meet some relatively easy criteria, so
+don't pay unless you really have to. Some banks will require you to have a social
+security number, but if you are being paid for your internship, your employer should
```
Comment (SolalPirelli) [inline _life/money-americans.md:285]: FWIW, at least as of 2017 First Tech didn't require an SSN to open an account but did to have online banking

File: _life/money-americans.md [hunk 3]
```diff
@@ -34,6 +34,7 @@ Kyle Matoba, Sepehr Elahi, Solal Pirelli, and others for helpful comments._
         -   [US state taxes](#us-state-taxes)
 		-   [US taxes for nonresident aliens](#us-taxes-for-nonresident-aliens)
         -   [Swiss taxes](#swiss-taxes)
+    - [Internships in the US for non-citizens](#internships-in-the-us-for-non-citizens)
```
Comment (SolalPirelli) [inline _life/money-americans.md:37]: worth linking to from the normal taxes page as well?

**Pull request merged**